Assets/MixedRealityToolkit.Extensions/Webrtc/Scripts/Signaling/ISignaler.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+
+namespace Microsoft.MixedReality.Toolkit.Extensions.Webrtc.Signaling
+{
+    public interface ISignaler
+    {
+        /// <summary>
+        /// Event that occurs when the signaler receives a new message
+        /// </summary>
+        event Action<SignalerMessage> OnMessage;
```

Comment:
We need a connected/disconnected event here, so we can track the signaling connection state. As part of 'connected' event Ice server list and Ice credentials may come too.